### PR TITLE
 Use Marathon's AWS Account to launch test clusters. 

### PR DESCRIPTION
**Pull request by jeschkies**
Summary:
This migrates our system integration tests to our own AWS account. It will also use temporary
SSH keys instead of a fixed pair.

Additionally cluster launch failures are marked as stable again. There was a bug before.

JIRA issues: MARATHON_EE-1802

### CODE REVIEW

File: ci/si_pipeline.sh [hunk 1]
```diff
@@ -48,16 +53,19 @@ function download-diagnostics-bundle {
...
 case $CLUSTER_LAUNCH_CODE in
   0)
       cp -f "$DOT_SHAKEDOWN" "$HOME/.shakedown"
       (cd tests && make init test)
       SI_CODE=$?
-      if [ ${SI_CODE} -gt 0 ]; then
-        download-diagnostics-bundle
-      fi
+      # if [ ${SI_CODE} -gt 0 ]; then
+      #  download-diagnostics-bundle
```
[the remark this answers is not in the document]
Comment (alenkacz) [reply ci/si_pipeline.sh]: nooo, my precious diagnostics bundle issue! :)

File: ci/launch_cluster.sh [hunk 2]
```diff
@@ -50,5 +47,8 @@ if ! ./dcos-launch wait; then
   exit 3
 fi
 
+# Extract SSH key
+jq -r .ssh_private_key cluster_info.json > "$CLI_TEST_SSH_KEY"
```
Comment (alenkacz) [inline ci/launch_cluster.sh]: I don't see the CLI_TEST_SSH_KEY used anywhere in this script so what is the point of this code?

File: ci/si_pipeline.sh [hunk 3]
```diff
@@ -48,16 +53,19 @@ function download-diagnostics-bundle {
 }
 
 # Launch cluster and run tests if launch was successful.
-export DCOS_URL=$( ./ci/launch_cluster.sh "$CHANNEL" "$VARIANT" | tail -1 )
+CLI_TEST_SSH_KEY="$(pwd)/$DEPLOYMENT_NAME.pem"
```
Comment (alenkacz) [inline ci/si_pipeline.sh]: how does this pem file get created?

File: ci/si_pipeline.sh (hunk 1 above)
[the remark this answers is not in the document]
Comment (zen-dog) [reply ci/si_pipeline.sh]: noooooo

File: ci/si_pipeline.sh (hunk 3 above)
Comment (jeschkies) [reply ci/si_pipeline.sh]: By `launch_cluster.sh:51`.

File: ci/launch_cluster.sh (hunk 2 above)
Comment (alenkacz) [reply ci/launch_cluster.sh]: oh, I get it now. Thanks!

**Pull request merged**